pytest_elk_reporter.py

```diff
@@ -192,8 +205,13 @@ def __init__(self, config):
         self.is_slave = False
 
     @property
-    def es_auth(self):
-        return self.es_username, self.es_password
+    def es_auth_args(self) -> dict[str, Any]:
```

Comment:
IIUC, this function always returns a `dict[str, dict[str, str]]`. if that's the case, probably we can just use `dict[str, dict[str, str]]` here?

Reply:
it can return `dict[str, tuple[str]]`, but is not important, it can return any request parameter and I don't want to document it all